crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -956,6 +949,15 @@ where
     }
 }
 
+impl<T> Remove<T> {
+    fn new(entity: Entity) -> Self {
```

Comment:
Doesn't this need to be pub for it to be useful?

Reply:
Correct :)